app/components/course-page/course-stage-step/second-stage-instructions-card/read-instructions-step.hbs

```diff
@@ -9,9 +9,16 @@
   </p>
   <ul>
     <li>
-      The
-      <LinkTo @route="course.stage.code-examples">Code Examples</LinkTo>
-      tab, which contains code examples from other users.
+      {{#if @isShowingLanguageGuide}}
+        The
+        <a href="#language-guide-card">{{this.currentLanguageName}} Guide</a>
```

Comment:
```suggestion
        <a href="#language-guide-card">"{{this.currentLanguageName}} Guide"</a>
```

(consistent with the "Your Task" quoting above)

Reply:
@andy1li Should I fix these too? (It seems that "Your Task" is the odd one here.)

<img width="825" alt="image" src="https://github.com/user-attachments/assets/4fd821e6-c63a-438c-b753-e2a248f80b8a">

Reply:
> It seems that "Your Task" is the odd one here.

Ah, nice catch - yeah okay let's remove the quotes from Your Task then! Could look weird to have a lot of quoted stuff there.